### ISSUE TITLE
{Pause(25000)} not working

### ISSUE THREAD

**Issue by Tacyon62** [EDCD/EDDI#504]
## EDDI version in which issue found
v3.0.0-b1 standalone.

## Steps to reproduce: Unknown if repeatable on other's installs

1. Using stock script for Docking Granted which calls Landing Pad Report. The only change that has been working for some time now was an increase to the "pause" statement in Docking Granted to 25000.
Ivona Amy TTS voice.

## Expected
With the {Pause(25000)} the station call out of cleared to pad "Y" Amy had been waiting till that was done and then announcing the pad position per the Landing Pad Report.

## Observed
The instant that the pad number declaration is on screen, Amy reads her Landing Pad Report talking over the stations call out of cleared to pad "Y". Note, this had been working for some time now. This is a fairly recent development.

## Investigation
* Increased {Pause(25000)} to {Pause(35000)} or {Pause(5000)} with no effect detected.
* Disable phonetic speech on or off, with no effect detected.
* "Compared to default" on both the Docking Granted trigger and Landing Pad Report reveals no differences other than the pause cmd value.

**Comment by Tkael** [comment]
I also use Ivona Amy and I don't have this problem (the disable phonetic speech button is unchecked in my config).
If you reset your script to default, does the pause work as expected? If so, your modified script may be flawed.

If it still fails to work with the default script, please
1. Open EDDI
2. Click the `Enable verbose logging button` in the `EDDI` tab
3. Navigate to the `Speech responder` tab, then open and `Test` any script with a pause function
4. Navigate back to the `EDDI` tab and click `Report an Issue`
5. A truncated version of your log should appear zipped on your desktop. 
6. Disable verbose logging
7. Attach the log file to this issue for review.

**Comment by Tacyon62** [comment]
Yea... I just noticed that the "disable phonetic speech" was enabled and don't recall setting it and since it was set, assumed that I'd flipped it to test for this issue. Sucks getting old.
Disabled the disable phonetic speech and the pause is working as expected. Hint: DOH, its says right next to it that it'll disable all SSML speech tags .. I'm like, "wait that sounds like a bad thing in this case" and it turned out to be.
Thanks for the quick heads up.
Closing "issue".
